setup/tue-env-config.bash

```diff
@@ -90,17 +126,17 @@ else
         functions=${functions//tue-env-/}
         # shellcheck disable=SC2086
         functions=$(echo $functions | tr ' ' '|')
-        while [ "$1" != "" ]
-        do
-            eval "
-                case $1 in
-                    $functions)
-                        tue-env-$1 ;;
-                    * )
-                        echo -e '[tue-env](config) Unknown config command: $1'
-                        exit 1 ;;
-                esac"
-            shift
-        done
+
+        cmd=$1
+        shift
+
+        eval "
+            case $cmd in
+                $functions )
+                        tue-env-$cmd $*;;
+                * )
+                    echo -e '[tue-env](config) Unknown config command: $cmd'
```

Comment:
Does the autocomplete support the one command rule.

Reply:
Yup.

Reply:
Or the autocomplete had a bug...

But I support the choice to only have command per call.